app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -190,16 +162,18 @@ function ReviewIssuesStep({
       />
 
       {!isFailedDocType && captureHints && (
-        <>
-          <p className="margin-bottom-0">{t('doc_auth.tips.review_issues_id_header_text')}</p>
-          <ul>
-            <li>{t('doc_auth.tips.review_issues_id_text1')}</li>
-            <li>{t('doc_auth.tips.review_issues_id_text2')}</li>
-            <li>{t('doc_auth.tips.review_issues_id_text3')}</li>
-            <li>{t('doc_auth.tips.review_issues_id_text4')}</li>
-          </ul>
-        </>
+        <TipList
+          title={t('doc_auth.tips.review_issues_id_header_text')}
+          items={[
+            t('doc_auth.tips.review_issues_id_text1'),
+            t('doc_auth.tips.review_issues_id_text2'),
+            t('doc_auth.tips.review_issues_id_text3'),
+            t('doc_auth.tips.review_issues_id_text4'),
+          ]}
+          translationNeeded={false}
```

Comment:
This is the default in the component, so isn't necessary. There's an argument for including it for clarity, my preference is to remove it 🤷🏼‍♂️. Same comment for the `TipList` in `documents-step.jsx`.